apps/web/src/components/AddressTile/AddressTileIcon.tsx

```diff
@@ -23,18 +23,32 @@ export const AddressTileIcon = memo(
     const account = getAccount(addressKind.pkh);
     const color = useColor();
 
+    const sizes = {
+      xs: 20,
```

Comment:
are you sure we don't need here values in px? just numbers

Reply:
yes, since this `size` prop is used only to adjust canvas icon size and the type is `number`